keyboards/crkbd/keymaps/armand1m/crkbd-base-layer.json

```diff
@@ -0,0 +1,281 @@
+[
```

Comment:
Could you remove this?

Reply:
I'd like to be able to keep it so I can import it back into the keyboard layout editor and update this on any layout changes. Do you feel it is unnecessary to keep it versioned?

Reply:
We generally don't allow KLE JSON files in the repo, it adds bloat for no good reason as nothing in QMK uses it.

Reply:
Thanks for the explanation! I'll take it out then. I'll store then on some other repo and just link in the README instead

Reply:
Removed all KLE JSON files